README.md

```diff
@@ -56,15 +55,15 @@ To add encryption certificates to the CMS:
 ### Adding sender certificates for digitally signing emails
 This module supports digital signing of emails, which provides further assurance for the recipient of where the email comes from.
 
-To sign an email you need both the .crt and private .key files. To upload these into the CMS:
+To sign an email you need both the certificate .pem and private key .pem files. To upload these into the CMS:
```

Comment:
`.pem` file reference updates

Reply:
Does the screenshot need to be updated too?

Reply:
Nah that current one displays the correct file formats, encryption = `.crt`